hooks/useGroupedTeamMemberData.tsx

```diff
@@ -20,8 +20,24 @@ export const groupedTeamMembersQuery = gql`
   }
 `
 
+interface GroupedTeamMembersQuery {
```

Comment:
❓ So did this come from gql/mongo and can it be dynamic, or did we copy it (still kinda confused on how the mongo gql typescript typing integration works)

Reply:
I wish it came from gql :sob: 
The end plan is to have these autogenerated but I generated this by hand from the gql schema.

Reply:
LOL and what do we as devs need to do to generate stuff from gql?

Reply:
Ahhhhh that one's with @EliasPinno right now.

This is a tutorial for the most basic graphql front end typegen setup [link](https://the-guild.dev/graphql/codegen/docs/getting-started). Just the front end part, you can disregard the backend part because that's already set up. Basically you the library downloads the graphql schema and then it generates types for all your different queries.

Not a big deal but once that's done I can swap out this type for the autogenerated one. I'm 100% sure the autogenerated type will be basically the exact same as this one hahaha.